src/app/pages/apps/components/chart-wizard/chart-wizard.component.html

```diff
@@ -27,7 +27,7 @@
           ixTest="save"
           [disabled]="form.invalid || isLoading"
         >
-          {{ 'Save' | translate }}
+          {{ isNew ? 'Install' : 'Update' | translate }}
```

Comment:
Message extractor won't be picked from this. It should be `isNew ? ('Install' | translate) : ('Update' | translate)`